NC_inference_plot.py

```diff
@@ -77,17 +78,26 @@ def inference(model, sound, sr=params.SAMPLE_RATE, result_1D=True, plot=False, )
     if result_1D:
         scores = np.sum(scores, axis=0)
 
-    return scores
+    return scores, stft_mag, stft_phase
```

Comment:
No documentational comment for `stft_mag` and `stft_phase`

Reply:
These two variables are the magnitude and phase of the STFT spectrogram. I pass them for DDAE to denoise and inverse transform the spectrogram back to the time domain.